### ISSUE TITLE
InternalServerError During ImportRequest of bacpac from Azure Storage

### ISSUE THREAD

**Issue by kine-mcorb** [Azure/azure-sdk-for-python#18649]
Packages:
azure-identity==1.5.0
azure-mgmt-sql==1.0.0

OS : Windows 10

Python 3.9

When trying to create a new Azure SQL Database it will fail at return the error:
```
Traceback (most recent call last):
  File "<path>\db_creator.py", line 47, in <module>
    database = sql_client.databases.begin_import_method(
  File "<path>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\azure\mgmt\sql\operations\_databases_operations.py", line 141, in begin_import_method
    raw_result = self._import_method_initial(
  File "<path>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\azure\mgmt\sql\operations\_databases_operations.py", line 94, 
in _import_method_initial
    raise HttpResponseError(response=response, error_format=ARMErrorFormat)
azure.core.exceptions.HttpResponseError: (InternalServerError) There was an internal server error that occurred during this request.
```

Code used:

```
from azure.identity import  ClientSecretCredential, AzureCliCredential
from azure.mgmt.sql.models import ImportRequest
from azure.mgmt.sql import SqlManagementClient

#Setup for SQL
database_name = 'dev_create' #set name of new database
storage_uri = "<bacpac_uri>" #provide url of bacpac in Azure Storage
storage_key = "<storage_key>" #Azure Storage Access Key
admin_login = "<sa>" # Azure SQL Server Admin
admin_pwd = "<sa_pswd>" # Azure SQL Server Admin password

resource = "<resource>"
server = "<server>"

#Setup for Client
tenant_id = "<tenant>"
client_id = "<client_id>"
client_secret = "<client_secret>"

subscription_id = "<sub_id>"

import_request = ImportRequest(
    storage_key_type='StorageAccessKey',
    storage_key=storage_key,
    storage_uri=storage_uri,
    administrator_login=admin_login,
    administrator_login_password=admin_pwd,
    database_name=database_name,
    edition='Basic',
    service_objective_name='Basic',
    max_size_bytes='2 GB',
    authentication_type='SQL')

# Try with Application Client Credentials
sql_client_credentials = ClientSecretCredential(
    tenant_id=tenant_id,
    client_id=client_id,
    client_secret=client_secret)

sql_client = SqlManagementClient(
    credential=sql_client_credentials,
    subscription_id=subscription_id)

database = sql_client.databases.begin_import_method(
    resource_group_name=resource,
    server_name=server,
    parameters=import_request)

# Try with Development CLI
cli_credential = AzureCliCredential()

sql_client = SqlManagementClient(
    credential=cli_credential,
    subscription_id=subscription_id)

database = sql_client.databases.begin_import_method(
    resource_group_name=resource,
    server_name=server,
    parameters=import_request)

```

I had this working with an older version of the Python SDK (azure-mgmt-sql==0.14.0 & azure-identity==1.0.0b4). I have tried using both Application credentials and using the CLI (I have confirmed my credentials are in the ServerAdmin group for the server). 

```
az version
{
  "azure-cli": "2.23.0",
  "azure-cli-core": "2.23.0",    
  "azure-cli-telemetry": "1.0.6",
  "extensions": {}
}
```

I have also successfully tested creating a new database by importing the bacpac from Azure storage using SSMS logged in as the SQL server Admin, with the same edition, service objective and size.

Any help or troubleshooting ideas appreciated.

**Comment by kine-mcorb** [comment]
I can achieve my desired result in two steps:

First create the empty database:
`sql_client.databases.begin_create_or_update(...)`

Then import the bacapc into the new database
```
from azure.mgmt.sql.models import ImportExtensionRequest
bacpac = ImportExtensionRequest(...)
sql_client.databases.begin_create_import_operation(...)
```

I'd still be interested in what is going wrong with the `begin_import_method()`. [Docs](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-mgmt-sql/1.0.0/azure.mgmt.sql.operations.html#azure.mgmt.sql.operations.DatabasesOperations.begin_import_method) describe it as 'Imports a bacpac into a new database.'

**Comment by kine-mcorb** [comment]
@Karishma-Tiwari-MSFT Thanks for following up. I haven't tried the `begin_import_method()` since I was able to achieve my result using the 2-step method I described above.

But looking at it again fresh, I can probably see the problem is I's not using the `ImportExistingDatabaseDefinition` class, which must've been a change between the older version and `azure-mgmt-sql==1.0.0`.

These docs say the `parameter` is an `ImportRequest` class: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-mgmt-sql/1.0.0/azure.mgmt.sql.operations.html#azure.mgmt.sql.operations.DatabasesOperations.begin_import_method

Whereas these docs have it as an `ImportExistingDatabaseDefinition`:
https://docs.microsoft.com/en-us/python/api/azure-mgmt-sql/azure.mgmt.sql.operations.databasesoperations?view=azure-python#begin-import-method-resource-group-name--server-name--database-name--parameters----kwargs-
